### DISCUSSION TITLE
How to open a JSON file in assets.1.0

### DISCUSSION THREAD

**Discussion by alexmathew11** [zepp-health/zeppos-forum#77] [Q&A]
I just wanted to put a button from which it can go to json file in assets . And please note that JSON file contains only text which wanted to display by pressing button . 

Thanks please give me

**Comment by juan518munoz** [comment]
Accessing files that are not images is done with the `hmFS` API.

https://docs.zepp.com/docs/1.0/reference/device-app-api/hmFS/read/
